CMakeLists.txt

```diff
@@ -258,12 +258,28 @@ add_subdirectory(src/app_demo)
 
 add_subdirectory(test)
 
+# expose to parent
+get_directory_property(hasParent PARENT_DIRECTORY)
+if(hasParent)
```

Comment:
what changes are needed for Gringofts users?

Reply:
grinKV has an example, include below librarise set